public/stylesheets/sass/_landing.scss

```diff
@@ -0,0 +1,74 @@
+.landing-hero {
+  background-image: url('../img/tricity-panorama.svg');
+  background-size: contain;
+  background-repeat: no-repeat;
+  background-position: bottom center;
+  height: 85vh;
+}
+
+.landing-community {
+  background-image: url('../img/bg-community.png');
+  height: 265px;
+}
+
+.landing-bg-rubies {
+  background-image: url('../img/rubies.svg');
+  background-repeat: no-repeat;
+  background-position-x: right;
+}
+
+.landing-link {
+  font-size: 1.1rem;
+}
+
+.landing-map-canvas {
+  width: 500px;
```

Comment:
I wonder  if we rather try to avoid px these days and use `vw` and `vh`, what do you think @angieelfin ???